### DISCUSSION TITLE
How to completely remove arc from Mac m1

### DISCUSSION THREAD

**Discussion by ghost** [crc-org/crc#3723] [Q&A]
Hi,
 I was using crc for learning on my laptop. Now I no longer require it and I did "crc stop, crc delete, crc cleanup" in this order. 
but am still able to use the crc command, there is no application running though and it has space consumed in my laptop.
So looking forward to some help to remove it completely.

**Comment by gbraad** [comment]
we do not remove the crc command, as you performed the download of this and
can be located anywhere on the system. in that case, you are required to
delete this yourself.

just to be sure, also do `rm -rf ~/.crc`, but the cleanup should be
sufficient.

On Sun, Jun 18, 2023 at 11:53 PM yo-art ***@***.***> wrote:

> Hi,
> I was using crc for learning on my laptop. Now I no longer require it and
> I did "crc stop, crc delete, crc cleanup" in this order.
> but am still able to use the crc command, there is no application running
> though and it has space consumed in my laptop.
> So looking forward to some help to remove it completely.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/crc-org/discussions/3723>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAAOZRN5XTWJQHBEJZZH6DXL4P7LANCNFSM6AAAAAAZK7AOG4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

   Gerard Braad | http://gbraad.nl

   STEM is the source, but BUILD is how you develop!
   [ Better Understanding Involves Learning and Doing ]

**Comment by ghost** [comment]
I tried checking, I am seeing three folders
1. .crc
2. .kube
3. .redhat

removed all 3 from ~ dir, but it reappeared when I did ls -ltra to recheck
